src/TorchSharp/Tensor/torch.BLAS_and_LAPACK_Operations.cs

```diff
@@ -0,0 +1,419 @@
+﻿// Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
```

Comment:
I love the code cleanup work, but I'm not in love with the name of this particular file. Is there something shorter without underscores we could use?

Reply:
I like to keep the naming aligned with the official PyTorch documentation. How about `torch.BlasAndLapackOperations`?